setup.cfg

```diff
@@ -0,0 +1,39 @@
+[metadata]
+name = icubam
+description = "ICU Bed Activity Monitor."
+version = 0.1.0
+url = https://github.com/icubam/icubam
+long_description = file: README.md
+license = Apache-2.0
+keywords = icu, covid-19, web server, map, dashboard
+classifiers =
+    Development Status :: 4 - Beta
+    Natural Language :: English
+    License :: OSI Approved :: Apache Software License
+    Programming Language :: Python
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.8
+    Intended Audience :: Healthcare Industry
+
+
+[options]
+install_requires =
```

Comment:
Either we keep it here, and remove `requirements.txt` or maybe there is an option to load `requirements.txt` here (or in setup.py).  I still think having a separate `requirements.txt` file is useful.

In either case, I would not keep both, as they are redundant and annoying to maintain in sync.

If we want to merge this quickly maybe let's remove it for now, and think about it in a follow up PR?

Reply:
@rth what I've seen done is the opposite way round i.e. keep dependencies in `setup.cfg` and replace the whole contents of `requirements.txt` by `-e .`. That way `pip install -r requirements.txt` and `pip install -e .` are equivalent and pull deps as well. How does that sound? Anyway I  agree that it's mostly nitpicking and can be done after this PR has been merged?

Reply:
That could also be fine, as long as we don't keep maintaining both. Though I'm not familiar with all deployment constraints.